pycc/cclambda.py

```diff
@@ -202,10 +202,11 @@ def r_L1(self, o, v, l1, l2, Hov, Hvv, Hoo, Hovvo, Hovov, Hvvvo, Hovoo, Hvovv, H
             r_l1 = r_l1 + contract('me,ieam->ia', l1, (2.0 * Hovvo - Hovov.swapaxes(2,3)))
             r_l1 = r_l1 + contract('imef,efam->ia', l2, Hvvvo)
             r_l1 = r_l1 - contract('mnae,iemn->ia', l2, Hovoo)
-            r_l1 = r_l1 - 2.0 * contract('ef,eifa->ia', Gvv, Hvovv)
-            r_l1 = r_l1 + contract('ef,eiaf->ia', Gvv, Hvovv)
-            r_l1 = r_l1 - 2.0 * contract('mn,mina->ia', Goo, Hooov)
-            r_l1 = r_l1 + contract('mn,imna->ia', Goo, Hooov)
+            if self.ccwfn.model == 'CCSD':
```

Comment:
These contractions will be needed for more than just CCSD, right?  Shouldn't they also be needed for CCD, CCSD(T), CC3, etc.?  In that case, would it be better to use a conditional that simply excludes them from CC2, since that's really the narrower case?